### ISSUE TITLE
Error when opening external folder

### ISSUE THREAD

**Issue by patschwork** [nextcloud/android#9713]
### Steps to reproduce
1. Open Android App
2. Navigate to external folder (cifs share) and open it
3. Crash

### Expected behaviour
- Folder shows content

### Actual behaviour
- ⬆️

### Environment data
Android version: 3.18.1

Device model: OnePlus 7T, Samsung Galxaxy Tab S7

Stock or customized system: Stock with extensions

Nextcloud app version: ?

Nextcloud server version: NC 20

Reverse proxy: no

#### Crash message (StackTrace)
```
************ CAUSE OF ERROR ************

java.lang.IllegalStateException: Couldn't read row 882, col 0 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.
	at android.database.CursorWindow.nativeGetLong(Native Method)
	at android.database.CursorWindow.getLong(CursorWindow.java:542)
	at android.database.AbstractWindowedCursor.getLong(AbstractWindowedCursor.java:77)
	at com.owncloud.android.providers.FileContentProvider.deleteDirectory(FileContentProvider.java:190)
	at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:146)
	at com.owncloud.android.providers.FileContentProvider.deleteDirectory(FileContentProvider.java:195)
	at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:146)
	at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:118)
	at android.content.ContentProvider.delete(ContentProvider.java:1820)
	at android.content.ContentProviderOperation.applyInternal(ContentProviderOperation.java:389)
	at android.content.ContentProviderOperation.apply(ContentProviderOperation.java:352)
	at com.owncloud.android.providers.FileContentProvider.applyBatch(FileContentProvider.java:694)
	at android.content.ContentProvider.applyBatch(ContentProvider.java:2476)
	at android.content.ContentProvider$Transport.applyBatch(ContentProvider.java:421)
	at android.content.ContentProviderClient.applyBatch(ContentProviderClient.java:586)
	at android.content.ContentProviderClient.applyBatch(ContentProviderClient.java:574)
	at android.content.ContentResolver.applyBatch(ContentResolver.java:2234)
	at com.owncloud.android.datamodel.FileDataStorageManager.saveFolder(FileDataStorageManager.java:449)
	at com.owncloud.android.operations.RefreshFolderOperation.synchronizeData(RefreshFolderOperation.java:533)
	at com.owncloud.android.operations.RefreshFolderOperation.fetchAndSyncRemoteFolder(RefreshFolderOperation.java:409)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:239)
	at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:363)
	at java.lang.Thread.run(Thread.java:923)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30180190
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: OnePlus
Device: OnePlus7TPro
Model: HD1913
Id: RKQ1.201022.002
Product: OnePlus7TPro_EEA

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: 2112102031

```

**Comment by patschwork** [comment]
I didn't found a similar error or any kind of solution.
The folder contains many photos. I also thought about, if it could be a timeout problem....(?)

**Comment by AlvaroBrey** [comment]
This is the same issue as #5926. Will (hopefully) be fixed soon by #9694
